OpenSim/Tools/Test/testVisualization.cpp

```diff
@@ -101,6 +103,9 @@ int main()
     try {
         LoadOpenSimLibrary("osimActuators");
         testVisModel("BuiltinGeometry.osim");
+        std::string testAppearanceModelFile = "AppearanceTest.osim";
+        createModel4AppearanceTest(testAppearanceModelFile);
```

Comment:
I think we could easily avoid writing out more model files unnecessarily by making `testVisModel()` take a model and make `createModel4AppearanceTest()` return a Model. It wouldn't preclude using models from file, you would just load em up prior to sending to `testVisModel()` and would allow us to add more visualization tests/models without creating a litany of files as proposed in #1151. We should beef up serialization tests to verify serialization as per #1150.